corehq/apps/hqwebapp/static/hqwebapp/js/lib/jquery-ui/multiselect/ui.multiselect.js

```diff
@@ -138,6 +138,18 @@ $.widget("ui.multiselect", {
 
 		$.widget.prototype.destroy.apply(this, arguments);
 	},
+	refresh: function() {
+		this.container.remove();
+		this._create();
+	},
+	addSelected: function(value, text) {
+		this.element.append(
+			$("<option/>").prop('value', value)
+			              .prop('selected', 'selected')
+			              .text(text)
+		);
+		this.refresh();
+	},
```

Comment:
This thing isn't even an official part of jquery-ui, and support for it has been dropped.  The spec for this original feature (assigning users to a location) had it be the same UI as we use for assigning users to a group.  I think if we were to change this one, we should change both (and any other instances of that widget).